main_process.py

```diff
@@ -0,0 +1,42 @@
+import cv2
+from project_types import NumpyArray
+from spatial_processing import blur_and_downsample_video
+from temporal_processing import frequency_filter_video
+from video_helpers import convert_video_to_np_array, convert_np_array_to_video, \
+    get_width_and_height_from_frames
+import numpy as np
+
+
+def merge_with_processed(original_frames: NumpyArray, processed_frames: NumpyArray, alpha: float,
+                         chrome_attenuation: float) -> NumpyArray:
+    """Merge the original frames of the video with the processed frames (spatially and temporaly).
+    Alpha and chrome attenuation are used to prettify the merge of the 2 array of frames."""
+    dest_width, dest_height = get_width_and_height_from_frames(original_frames)
+    images = []
+    for index, frame in enumerate(processed_frames):
+        # Amplification
+        h, s, v = cv2.split(frame)
+        h *= alpha
+        s *= alpha * chrome_attenuation
+        v *= alpha * chrome_attenuation
+        amplified_frame = cv2.merge((h, s, v))
+        # Merging
+        resized_amplified_frame = cv2.resize(amplified_frame, (dest_width, dest_height))
+        original_frame = original_frames[index]
+        merged = cv2.addWeighted(original_frame, 1, resized_amplified_frame, 0.3, 0)
```

Comment:
adding two hsv colors together will probably create a monster, also maybe we should see it as an average color by using the coefficients 0.7 and 0.3 (rather than 1 and 0.3)